lib/wibox/layout/free.lua

```diff
@@ -0,0 +1,215 @@
+---------------------------------------------------------------------------
+-- A layout with widgets added at specific positions.
+--
+-- Use cases include desktop icons, complex custom composed widgets, a floating
+-- client layout and fine grained control over the output.
+--
+--@DOC_wibox_layout_defaults_free_EXAMPLE@
+-- @author Emmanuel Lepage Vallee
+-- @copyright 2016 Emmanuel Lepage Vallee
+-- @release @AWESOME_VERSION@
+-- @classmod wibox.layout.free
+---------------------------------------------------------------------------
+local gtable = require("gears.table")
+local base = require("wibox.widget.base")
+
+local free_layout = {}
+
+--- Add some widgets to the given stack layout
```

Comment:
Maybe he meant "free" as in "freedesktop"? Because that seems rather appropriate to me.